server/package.json

```diff
@@ -25,8 +26,31 @@
     "web-tree-sitter": "^0.20.8"
   },
   "scripts": {
+    "prepack": "cd .. && [ -d client/resources ] || scripts/fetch-docs.sh && cd server && cp -r ../client/resources ../client/tree-sitter-bitbake.wasm . && tsc --outDir out",
     "installServer": "installServerIntoExtension ../client ./package.json ./tsconfig.json"
   },
+  "devDependencies": {
+    "@types/glob": "^8.1.0",
+    "@types/jest": "^29.5.5",
+    "@types/mocha": "^10.0.1",
+    "@types/node": "^20.6.1",
+    "@types/vscode": "^1.75.1",
+    "@vscode/test-electron": "^2.3.5",
```

Comment:
Probably don't need vsce, test-electron, ... to compile. However some vscode APIs might still be usefull. Could you try to eliminate what's not needed from this list by dichotomy? Thanks